meson.build

```diff
@@ -524,11 +524,6 @@ if libsystemd.found()
   endif
 endif
 
-elogind = dependency('systemd', 'libelogind', required: get_option('elogind'))
```

Comment:
this one instead of dropping, let's just discover if elogind is found but not make it mandatory.  Then we have no complexity but distros that care about elogind can still use it.

Reply:
Do you know offhand who cares about elogind?

Reply:
It was introduced a long time ago in de2b70bc7d specifically for Gentoo with OpenRC.
CC @marcin-sucharski

Reply:
I think it's okay to say we don't support it upstream; I'd be fine if gentoo supported it downstream with a patch. What I'm trying to avoid is being in a situation where we rely on systemd for some really important security lockdown (so that the service is secure) and we forget about the people using not-systemd. At the moment people using OpenRC are much more exposed when using fwupd.

Reply:
Got it.  In that case I'm OK with us dropping it.